web/src/components/molecules/Common/Header/index.tsx

```diff
@@ -65,7 +66,11 @@ const HeaderMolecule: React.FC<Props> = ({
           children: workspaces
             ?.filter(workspace => workspace.id === personalWorkspace?.id)
             ?.map(workspace => ({
-              label: <MenuText>{workspace.name}</MenuText>,
+              label: (
+                <Tooltip title={workspace.name} placement="right">
+                  <MenuText>{workspace.name}</MenuText>
```

Comment:
Is there any point making a tooltip that just shows what the menu text already shows? Typically a tooltip would have a description or something different than the original, no?

Reply:
Yup, because the workspace name can be very long (it will end with three dots), so I've been asked to have a tooltip for the name.